src/spikeinterface/core/job_tools.py

```diff
@@ -60,7 +60,7 @@
 
 
 def fix_job_kwargs(runtime_job_kwargs):
-    from .globals import get_global_job_kwargs
+    from .globals import get_global_job_kwargs, global_job_kwargs_set
```

Comment:
```suggestion
    from .globals import get_global_job_kwargs, is_set_global_job_kwargs_set
```

Reply:
The use of the function better because it ensure that it is a global.

Reply:
Why? I would love to learn the difference here between checking with the function and just importing the boolean. I don't really use `global` myself.

Reply:
I agree with Zach here. It's the same and it's super internal

Reply:
because it ensure that this import is not masked in between by another variable with the same name.
globals in python are not super garanty unless you put global before.

Reply:
globals corss moules is evill in python having a function interface is more secure.

Reply:
Good to know. Thanks :)

Reply:
done in last commit